sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/README.md

```diff
@@ -78,7 +84,33 @@ Assembly properties required for running unit tests.
 
 ## Troubleshooting
 
-Coming soon.
+All Time Series Insights service operations will contain helpful error information in response [TimeSeriesOperationError](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Generated/Models/TimeSeriesOperationError.cs). 
```

Comment:
@barustum Does this look right?

Reply:
I wouldn't say 'All'. Reason being, some APIs do not return error or result. For example, GetAsync for Hierarchies client return TimeSeriesHierarchy. 

Maybe something along these lines:
Time Series Insights service operation failures are usually returned to the user as [TimeSeriesOperationError](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Generated/Models/TimeSeriesOperationError.cs). The TimeSeriesOperationError response is either returned directly by the client library API, or as a nested property within the actual response for the client library API. For example, the DeleteByName API that is part of the hierarchies client returns a TimeSeriesOperationError directly. Whereas, the Replace API that is part of the instances client returns a InstancesOperationResult, which has a TimeSeriesOperationError property nested within it.